api/src/controllers/activity.ts

```diff
@@ -68,54 +66,7 @@ router.get(
 	respond,
 );
 
-const createCommentSchema = Joi.object({
-	comment: Joi.string().required(),
-	collection: Joi.string().required(),
-	item: [Joi.number().required(), Joi.string().required()],
-});
-
-router.post(
-	'/comment',
-	asyncHandler(async (req, res, next) => {
-		const service = new ActivityService({
-			accountability: req.accountability,
-			schema: req.schema,
-		});
-
-		const { error } = createCommentSchema.validate(req.body);
-
-		if (error) {
-			throw new InvalidPayloadError({ reason: error.message });
-		}
-
-		const primaryKey = await service.createOne({
-			...req.body,
-			action: Action.COMMENT,
-			user: req.accountability?.user,
-			ip: getIPFromReq(req),
-			user_agent: req.accountability?.userAgent,
-			origin: req.get('origin'),
-		});
-
-		try {
-			const record = await service.readOne(primaryKey, req.sanitizedQuery);
-
-			res.locals['payload'] = {
-				data: record || null,
-			};
-		} catch (error: any) {
-			if (isDirectusError(error, ErrorCode.Forbidden)) {
-				return next();
-			}
-
-			throw error;
-		}
-
-		return next();
-	}),
-	respond,
-);
-
+// TODO: Remove legacy commenting in upcoming version
```

Comment:
If we are leaving patch and delete for backwards compatible (per discussion with Ian) we should leave post as well. This would ensure all comment operation work while migration is taking place. We can remove these in a subsequent version once we are sure migrations are no longer a consideration